### PR TITLE
Feature/core 702 sharing

### PR DESCRIPTION
**Pull request by Tschuck**
- fix wrong translations
- restrict sharing side panel entries

### CODE REVIEW

File: dapps/onboarding.vue/src/components/sign-up/sign-up.ts [hunk 1]
```diff
@@ -128,6 +126,58 @@ export default class SignUp extends mixins(EvanComponent) {
...
+    // add finishing step
+    steps.push({
+      title: '_onboarding.sign-up.steps.captcha.title',
+      disabled: () => {
+        if (creatingOrOnboarded()) {
+          return true;
+        }
+
+        switch (this.profileForm.accountType.value) {
+          case 'company': {
```
Comment (datrinh) [inline dapps/onboarding.vue/src/components/sign-up/sign-up.ts:168]: You could omit curly brackets in switch/case blocks. Saves some lines :)

File: dapps/onboarding.vue/src/components/sign-up/sign-up.ts [hunk 2]
```diff
@@ -143,17 +193,19 @@ export default class SignUp extends mixins(EvanComponent) {
...
       alias: {
         value: '',
         validate: function(vueInstance: SignUp, form: ProfileFormInterface) {
           return this.value.length !== 0;
         },
-        uiSpecs: { attr: { hint: true, required: true, } },
+        uiSpecs: {
+          attr: {
+            hint: true,
+            required: true,
```
Comment (datrinh) [inline dapps/onboarding.vue/src/components/sign-up/sign-up.ts:205]:
```suggestion
            required: true
```

File: dapps/onboarding.vue/src/components/sign-up/sign-up.ts [hunk 3]
```diff
@@ -143,17 +193,19 @@ export default class SignUp extends mixins(EvanComponent) {
...
+          },
+          label: () => this.$t(`_onboarding.sign-up.alias.${ this.profileForm.accountType.value }`),
```
Comment (datrinh) [inline dapps/onboarding.vue/src/components/sign-up/sign-up.ts:207]:
```suggestion
          label: () => this.$t(`_onboarding.sign-up.alias.${ this.profileForm.accountType.value }`)
```

File: dapps/profile.vue/src/components/verifications/notary/actions/request/request.ts [hunk 4]
```diff
@@ -24,8 +24,8 @@ import { Prop, Watch } from 'vue-property-decorator';
 
 // evan.network imports
 import { EvanComponent, EvanForm, EvanFormControl } from '@evan.network/ui-vue-core';
+import { bccUtils, } from '@evan.network/ui';
```
Comment (datrinh) [inline dapps/profile.vue/src/components/verifications/notary/actions/request/request.ts:27]:
```suggestion
import { bccUtils } from '@evan.network/ui';
```

File: dapps/profile.vue/src/components/verifications/notary/actions/request/request.ts [hunk 5]
```diff
@@ -43,6 +43,10 @@ interface LabeledEntry {
 }
 
 interface CompanyData {
+  accountDetails: {
+    accountName: string,
```
Comment (datrinh) [inline dapps/profile.vue/src/components/verifications/notary/actions/request/request.ts]:
```suggestion
    accountName: string;
```

File: dapps/profile.vue/src/components/verifications/notary/actions/request/request.ts [hunk 6]
```diff
@@ -43,6 +43,10 @@ interface LabeledEntry {
...
+    profileType: string,
```
Comment (datrinh) [inline dapps/profile.vue/src/components/verifications/notary/actions/request/request.ts]:
```suggestion
    profileType: string;
```

Comment (datrinh) [reply dapps/profile.vue/src/components/verifications/notary/actions/request/request.ts]: This is minor, but semicolons are better suited in interfaces.

File: dapps/profile.vue/src/components/verifications/notary/topic-display/topic-display.ts [hunk 7]
```diff
@@ -24,9 +24,8 @@ import moment from 'moment';
 
 // evan.network imports
 import { EvanComponent } from '@evan.network/ui-vue-core';
-import { FileHandler, } from '@evan.network/ui';
+import { FileHandler, bccUtils, } from '@evan.network/ui';
```
Comment (datrinh) [inline dapps/profile.vue/src/components/verifications/notary/topic-display/topic-display.ts:27]:
```suggestion
import { FileHandler, bccUtils } from '@evan.network/ui';
```

**Review by datrinh — CHANGES_REQUESTED** [review]

**Comment by datrinh** [comment]
> This pull request **introduces 3 alerts** and **fixes 2** when merging [bc269a2](https://github.com/evannetwork/ui-core-dapps/commit/bc269a2980b9ffa3e456941e2cc72505c5d03351) into [ffa9306](https://github.com/evannetwork/ui-core-dapps/commit/ffa9306cd7e79bb94ea9a2bbe520164292b1ce08) - [view on LGTM.com](https://lgtm.com/projects/g/evannetwork/ui-core-dapps/rev/pr-435e91013642532bc17713d27ff998cef21a126c)
> 
> **new alerts:**
> 
> * 3 for Unused variable, import, function or class
> 
> **fixed alerts:**
> 
> * 2 for Unused variable, import, function or class

Please fix those alerts too :)

**Review by datrinh — APPROVED** [review]

File: dapps/profile.vue/src/i18n/de.json [hunk 8]
```diff
@@ -126,20 +126,20 @@
...
-      "profile-update": "Profilinformationen werden gespeichert...",
+      "profile-share": "Identitätsinformationen teilen...",
+      "profile-type": "Identitätstyp wird geändert...",
+      "profile-update": "Identitätinformationen werden gespeichert...",
       "request-verification": "Verifikation wird beantragt...",
       "verification-accept": "Verifikation wird angenommen..."
     },
     "not-permitted": {
-      "accountDetails": "Profil-Informationen",
+      "accountDetails": "Identitätinformationen",
```
Comment (S3bb1) [inline dapps/profile.vue/src/i18n/de.json]:
```suggestion
      "accountDetails": "Identitätinformationen",
```
```suggestion
      "accountDetails": "Identitätsinformationen",
```

File: dapps/onboarding.vue/src/i18n/de.json [hunk 9]
```diff
@@ -192,29 +202,30 @@
...
+        "further-changes": "Änderungen sind nach der Erstellung möglich.",
         "steps": {
           "dbcp": {
-            "desc": "Wählen Sie einen Twin-Type und vergeben Sie einen sprechenden Namen",
+            "desc": "Wählen Sie einen Typ für Ihren Digitalen Zwilling und vergeben Sie einen Namen.",
             "title": "Generelle Informationen"
           },
           "finish": {
-            "desc": "Für Ihren Zugang zu Ihrem Digital Twin benötigen wir einen Alias und ein Passwort.",
+            "desc": "Für Ihren Zugang zu Ihrem Digitaler Zwilling benötigen wir einen Alias und ein Passwort.",
```
Comment (S3bb1) [inline dapps/onboarding.vue/src/i18n/de.json]:
```suggestion
            "desc": "Für den Zugang zu Ihrem Digitalen Zwilling benötigen wir einen Alias und ein Passwort.",
```

**Review by S3bb1 — CHANGES_REQUESTED** [review]
see comments

**Review by S3bb1 — APPROVED** [review]

**Pull request merged**